src/data/phase/dto.type.ts

```diff
@@ -20,11 +20,18 @@ export type TaskDTO = Readonly<{
   points: number
 }>
 
+export type ChallengeDTO = Readonly<{
+  group: string
+  challengeDescription: JSX.Element
+  tasks: TaskDTO[]
```

Comment:
Please, do not reuse directly the `TaskDTO` type for challenges part, they must be agnostic each other.

If it's the same, you can create an alias of type but I think you don't need for now the `points` property.